src/test/scala/org/allenai/pipeline/SamplePipeline.scala

```diff
@@ -53,7 +53,7 @@ class SamplePipeline extends UnitSpec
     val model = pipeline.Persist.Singleton.asJson(new TrainModel(trainDataPersisted), None, ".json")
     val measure: Producer[PRMeasurement] =
       pipeline.Persist.Collection.asText(new MeasureModel(model, testData), None, ".txt")
-    pipeline.run("SamplePipeline", measure)
+    pipeline.run("SamplePipeline")
```

Comment:
I like this. Just run the pipeline already, it knows what producers it needs to generate.